app/src/main/java/zegreanu/cristi/criminalintent/CrimeListFragment.kt

```diff
@@ -67,10 +67,18 @@ class CrimeListFragment : Fragment() {
 
     private class CrimeAdapter(private val crimes: List<Crime>) :
         RecyclerView.Adapter<CrimeViewHolder>() {
+        val requirePoliceViewType = 0
+        val notRequirePoliceViewType = 1
 
         override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CrimeViewHolder {
             val itemView =
-                LayoutInflater.from(parent.context).inflate(R.layout.list_item_crime, parent, false)
+                if (viewType == requirePoliceViewType)
+                    LayoutInflater.from(parent.context)
+                        .inflate(R.layout.list_item_crime_police, parent, false)
+                else
+                    LayoutInflater.from(parent.context)
+                        .inflate(R.layout.list_item_crime, parent, false)
+
             return CrimeViewHolder(itemView)
```

Comment:
Just as an FYI, we usually have different view holders as well, depending on the differences between the two layouts